src/pages/about-us/_OurPrinciples.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import styled from 'styled-components'
+import { our_principles } from './_data'
+import { localize } from 'components/localization'
+import { SectionContainer, Flex, Box } from 'components/containers'
+import { Header, Text } from 'components/elements'
+import device from 'themes/device'
+import { LinkButton } from 'components/form'
+
+const StyledSection = styled(SectionContainer)`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 80px 16px 120px;
+    background-color: var(--color-white);
+
+    @media ${device.tablet} {
+        padding: 40px 16px;
+    }
+
+    h2 {
+        margin: 0;
+        padding: 0;
+        @media ${device.laptop} {
+            font-size: 28px;
+            margin-bottom: 24px;
+        }
```

Comment:
Please use the `Header` component with its respective type instead of this.

Reply:
done